recipes/libsass/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "libsass" %}
+{% set version = "0.13.2" %}
+{% set sha256 = "f6270e6f5eb0bfa4e5ecf5b9e6443f576a6fe47cbb732b8d6acaf396c323ec0a" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    
+  run:
+    - python
+    - twine ==1.5.0
+    - six
+
+test:
+  import:
+    - libsass
+    
+about:
+  home: https://github.com/dahlia/libsass-python
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Can you add a comment referring to the PR I just opened, https://github.com/dahlia/libsass-python/pull/216, so that packaging the license here won't be necessary?

Reply:
Thanks. I will update the license file path to "LICENSE". I can also delete the LICENSE file from the repo, but it might cause some problems so I won't do that.

Reply:
No, you can't do that until there's a new libsass release that actually packages `LICENSE`...just add a comment that for the next release you can delete the `LICENSE` file from this repo and change it to just `LICENSE` then.